angular-portfolio/src/app/name/name.component.scss

```diff
@@ -0,0 +1,38 @@
+
+.name {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    margin-top: -5%;
+    margin-left: -20%;
+    box-sizing: border-box;
+    height: 25%;
+    width: 40%;
+    text-align: center;
+    background-color: rgb(47, 54, 64, 0.5);
+
+    border-radius: 2vw !important;
```

Comment:
I don't think you should use vw for the border radius. This will make the element "curvier" based on the size of the window, and I can't imagine when you would want that behavior. Also, be wary of using !important, usually there's a better way (although not always).